lib/di.dart

```diff
@@ -987,8 +997,12 @@ Future<void> setup({
           SettingsStoreBase.walletPasswordDirectInput,
         );
       case WalletType.litecoin:
-        return bitcoin!.createLitecoinWalletService(_walletInfoSource, _unspentCoinsInfoSource,
-            SettingsStoreBase.walletPasswordDirectInput);
+        return bitcoin!.createLitecoinWalletService(
+          _walletInfoSource,
+          _unspentCoinsInfoSource,
+          getIt.get<SettingsStore>().mwebAlwaysScan,
+          SettingsStoreBase.walletPasswordDirectInput,
```

Comment:
this should be reverted to match the parameters

Reply:
ah, I must've missed this when merging main